code/game/objects/items/tools/mining_tools.dm

```diff
@@ -257,6 +250,9 @@
 		spark_system.set_up(5, 0, M)
 		spark_system.attach(M)
 		spark_system.start(M)
+		cell.charge += 200
+		to_chat(world, span_warning("[src] violently vaporizes chunks of the Xenomorph upon contact!"))
+		to_chat(user, span_notice("The vaporized matter causes a temporary feedback loop in the battery! <b>Charge Amount: [cell.charge]/[cell.maxcharge]</b>"))
```

Comment:
Lmfao

You should know what to fix here

Reply:
You mean you don't want everyone to hear about the vaporizing Xenomorph? I think it's pretty cool

Reply:
Fixed